sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/statefulOperators.scala

```diff
@@ -111,6 +111,10 @@ trait StatefulOperator extends SparkPlan {
     }
   }
 
+  lazy val stateStoreEncoding: String =
```

Comment:
Could we add some validation tests for this config ?

Reply:
spoke offline - no need since SQLConf has validation